src/App.jsx

```diff
@@ -1,30 +1,26 @@
-import { BrowserRouter } from 'react-router-dom';
+import { BrowserRouter , Routes,Route} from 'react-router-dom';
 import Navbar from './components/NavBar/Navbar'
 import BookCard from '/src/components/BookCard/BookCard.jsx';
 import BookDetails from '/src/components/BookDetails/BookDetails.jsx';
 import booksData from '/src/data/booksData.js';
 const URL = 'http://localhost:8000/api/v1/';
 
 function App() {
-const cards = booksData.map((item) => {
-		return <BookCard key={item.id} {...item} />;
-	});
 
 	const books = booksData.map((item) => {
 		return <BookDetails key={item.id} {...item} />;
 	});
 
   return (
-    <>
+	<>
```

Comment:
Make sure to format all of these files before merging! I am once again advocating for "Format on Save" and "Autosave after delay" in your VS Code settings.